_layouts/events.html

```diff
@@ -39,7 +37,7 @@ <h3 class="alt text-gray">{{event.title}}</h3>
         {{event.venue}}
        </div>
        <div class="speaker">
-        {{event.speaker}}
+        {{event.speaker}}  is speaking
```

Comment:
The english here is incorrect if event.speaker has more than one name in it (e.g. "Tim & Dawn is speaking")